Fix PoolingOptionsTest#should_refresh_single_connected_host()

tolbertam opened the pull request:
Changed LimitingLoadBalancingPolicy to treat hosts passed in the constructor as chosen hosts and pass to its delegate LoadBalancingPolicy as hosts passed into the constructor this way are to be set to up after LoadBalancingPolicy#init is called in Cluster.Manager.init().

There was a change for JAVA-511 to make it so LoadBalancingPolicy#onUp is not called for contactPoints on initialization as it caused some extraneous warning messages (d0eacbb3f8f6ca093383773d4b770cd9d9e95661) and this integration test has been failing ever since.

pingtimeout commented:
I am not sure we have strict commit message rules.  But I think your commit would be more readable if it followed the usual conventions, something along the lines of [this](https://github.com/erlang/otp/wiki/writing-good-commit-messages).

tolbertam committed:
Fix PoolingOptionsTest#should_refresh_single_connected_host()

Changed LimitingLoadBalancingPolicy to treat hosts passed in the
constructor as chosen hosts and pass to its delegate
LoadBalancingPolicy as hosts passed into the constructor this way are
to be set to up after LoadBalancingPolicy#init is called in
Cluster.Manager.init().

tolbertam commented:
Thanks for the suggestion @pingtimeout :).  I've updated the commit message and repushed.
